src/components/gridListItem/index.tsx

```diff
@@ -85,6 +94,10 @@ export interface GridListItemProps {
    * Should content be align to start (default is center)
    */
   alignToStart?: boolean;
+  /**
+   * Content horizontal alignment (default is center)
+   */
+  horizontalAlignment?: ItemHorizontalAlignmentProp;
```

Comment:
Why do we need this prop? Why the user can't just pass 'alignItems' in the containerStyle?